firedrake/interpolation.py

```diff
@@ -102,15 +102,15 @@ def interpolate(
         defined on the source mesh. For example, where nodes are point
         evaluations, points in the target mesh that are not in the source mesh.
         When ``False`` this raises a ``ValueError`` should this occur. When
-        ``True`` the corresponding values are set to zero or to the value
+        ``True`` the corresponding values are left unchanged or set to the value
         ``default_missing_val`` if given. Ignored if interpolating within the
         same mesh or onto a :func:`.VertexOnlyMesh` (the behaviour of a
         :func:`.VertexOnlyMesh` in this scenario is, at present, set when
         it is created).
     :kwarg default_missing_val: For interpolation across meshes: the optional
         value to assign to DoFs in the target mesh that are outside the source
-        mesh. If this is not set then zero is used. Ignored if interpolating
-        within the same mesh or onto a :func:`.VertexOnlyMesh`.
+        mesh. If this is not set then the values are left unchanged. Ignored if
+        interpolating within the same mesh or onto a :func:`.VertexOnlyMesh`.
```

Comment:
Same here. It needs rewording to say what happens if you supply some `output=func`